pom.xml

```
@@ -191,7 +191,7 @@
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-bigtable</artifactId>
-            <version>1.15.0</version>
+            <version>1.21.3</version>
```

```suggestion
            <version>1.20.1</version>
```
There is an issue with 1.21.3